eclipse-collections-code-generator/src/main/java/org/eclipse/collections/codegenerator/EclipseCollectionsCodeGenerator.java

```diff
@@ -30,10 +30,12 @@
 {
     public static final String GENERATED_TEST_SOURCES_LOCATION = "target/generated-test-sources/java/";
     public static final String GENERATED_SOURCES_LOCATION = "target/generated-sources/java/";
+    public static final String GENERATED_SERVICES_LOCATION = "target/generated-sources/resources/META-INF/services";
```

Comment:
I was expecting to see `target/classes/META-INF/services` here. These files aren't sources (they don't get compiled).

Reply:
I tried generating here, but it gets wiped out in the build process, so I decided to mirror the checked in files in src\main\resources.

Reply:
The reason is that the code generator plugin currently only has a single target, with a single phase, generate-sources.

```java
/**
 * @goal generate
 * @phase generate-sources
 * @requiresDependencyResolution compile
 */
public class GenerateMojo extends AbstractMojo
{
```

So it's running too late to generate resources.